### DISCUSSION TITLE
immorality/cruelty/animal/

### DISCUSSION THREAD

**Discussion by giscus** [SageofPHY6Paths/hinduismdebunked.com#19] [Comments]
# immorality/cruelty/animal/

A website that debunks Hinduism

https://hinduismdebunked.com/immorality/cruelty/animal/

**Comment by SARGEHartman** [comment]
Hinduism also allows Bestiality i.e. sex with animals. Shiva had raped Agni whilst Agni was in the form a bird which is a case of bestiality, let me quote that passage again,
Brahma Purana, Gautami-Mahatmya 58.16-24 ”At the instance of Devas the fire-god assumed the form of a parrot and went to the place where the lord of the worlds was sporting with Uma…The lord of Devas called the fire-god moving about in front of him, in the form of a bird and said, ”O Agni, you have been recognised through various ways. Do not speak anything. Open your mouth. Take this in and carry it with you.” After saying this lord Sambhu discharged a great deal of semen into the mouth of fire-god. With this semen within him Agni could not at all go. Utterly tired, Agni sat down on the bank of Ganga. He then discharged a greater portion of that semen into Krttikas wherefrom Karttika was born. The remaining portion of the semen of Sambhu that had been within his body, Agni split into two and deposited into his beloved wife Svaha who was particularly eager to have a child.” Tr. Board of Scholars, Edited by J.L. Shastri

A passage from Mahabharata shows that a Rishi named Kindama used to have sex with other deer as he was uncomfortable in having sex with humans,

Mahabharata 1.118.5-38 “Vaisampayana said, ‘O king, one day Pandu, while roaming about in the woods (on the southern slopes of the Himavat) that teemed with deer and wild animals of fierce disposition, saw a large deer, that seemed to be the leader of a herd, serving his mate. Beholding the animals, the monarch pierced them both with five of his sharp and swift arrows winged with golden feathers. O monarch, that was no deer that Pandu struck at, but a Rishi’s son of great ascetic merit who was enjoying his mate in the form of a deer. Pierced by Pandu, while engaged in the act of intercourse, he fell down to the ground, uttering cries that were of a man and began to weep bitterly…[Rishi in deer form said] The time of sexual intercourse is agreeable to every creature and productive of good to all. O king [Pandu], with this my mate I was engaged in the gratification of my sexual desire. But that effort of mine hath been rendered futile by thee…I am, O king [Pandu], a Muni who liveth on fruits and roots, though disguised as a deer…I was engaged in sexual intercourse with this deer, because my feelings of modesty did not permit me to indulge in such an act in human society. In the form of a deer I rove in the deep woods in the company of other deer. Thou hast slain me without knowing that I am a Brahmana, the sin of having slain a Brahmana shall not, therefore, be thine. But senseless man, as you have killed me, disguised as a deer, at such a time, thy fate shall certainly be even like mine. When, approaching thy wife lustfully, thou wilt unite with her even as I had done with mine… Tr. Kisari Mohan Ganguli

Maharshi Manu writes that many sages were born after sages had sex with animals,

Manu Smriti 10.72 ”Since through their excellent energies (potency), seeds, cast in the wombs of beasts (by the holy sages), fructified in the shapes of human beings, who became honoured and commendable Rishis in life; the seed is commended (as of greater importance in an act of fecundation).” Tr. M.N. Dutt
